### PR TITLE
Add toasts and useToast documentation

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | We need to document toasts and useToast
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

## Quickview
image

### CODE REVIEW

**Comment by NeOMakinG** [comment]
@Hlavtox @mparvazi @SharakPL

Feel free to review if you want (you're not forced)

File: stories/theme/javascript/components/useToast/index.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  content: HTMLElement | null;
+  // Bootstrap methods wrapped
+  show: () => boolean;
+  hide: () => boolean;
+  dispose: () => boolean;
+  remove: () => boolean;
+  message: (markup?: string) => string | boolean;
+}
+```
+
```
Comment (NeOMakinG) [inline stories/theme/javascript/components/useToast/index.stories.mdx:47]: This is probably the most important part to review

**Comment by mparvazi** [comment]
dispose() will be useful when you just have a toast inside the DOM (without template):

<div class="toast-container">
    <div class="toast">...</div>
</div>

So when you call dispose() then you can create another toast instance with this div.

But in useToast we used template for cloning and appending in the DOM.
I mean when you call useToast you have a corresponding toast element in the DOM.

I think BS developers forget that something like <template> is more flexible so they defined something like dispose()

**Comment by NeOMakinG** [comment]
> dispose() will be useful when you just have a toast inside the DOM (without template):
> 
> ```
> <div class="toast-container">
>     <div class="toast">...</div>
> </div>
> ```
> 
> So when you call dispose() then you can create another toast instance with this div.
> 
> But in useToast we used template for cloning and appending in the DOM. I mean when you call useToast you have a corresponding toast element in the DOM.
> 
> I think BS developers forget that something like `<template>` is more flexible so they defined something like dispose()

At least, we leave the liberty to anyone, to use whatever they wants :)

**Comment by NeOMakinG** [comment]
This is not blocking anything, let's just merge this and improve it later if needed

**Pull request merged**